rebasehelper/output_tool.py

```diff
@@ -47,6 +49,10 @@ def get_report_path(cls, app):
     def get_extension(cls):
         raise NotImplementedError
 
+    @classmethod
+    def prepend_results_dir_short(cls, *path_members):
```

Comment:
What does `_short` mean?

Reply:
I wanted to highlight the fact, that it is not the full results dir path.

Reply:
By full you mean absolute? Anyway, if you must append anything, `prepend_results_dir_name` sounds better to me.